client/app/components/CreateEventModal.jsx

```diff
@@ -0,0 +1,181 @@
+import React, { PropTypes } from 'react';
+import { Button, Modal, FormGroup, ControlLabel, FormControl, Alert } from 'react-bootstrap';
+import DateTimeField from 'react-bootstrap-datetimepicker';
+import moment from 'moment';
+import axios from 'axios';
+import _ from 'lodash';
+
+const { func, bool, array } = PropTypes;
+
+const DATE_FORMAT = 'DD/MM/YYYY HH:mm';
+
+export default class CreateEventModal extends React.Component {
+  static propTypes = {
+    openModal:       func,
+    closeModal:      func,
+    showModal:       bool,
+    conferenceRooms: array
+  };
+
+  constructor() {
+    super();
+
+    const initialDate = moment().format(DATE_FORMAT);
+
+    this.state = {
+      showErrorMessage: false,
+      conferenceRoomId: 1,
```

Comment:
Hardcoding this is is probably not a great idea especially that ids could also be strings and they wouldn't necessarily start at 1.